### DISCUSSION TITLE
Cannot read properties of null (reading 'iterations')

### DISCUSSION THREAD

**Discussion by boodmi** [dani-garcia/vaultwarden#3339] [Q&A] [answered]
Bitwarden Inc. (Powered by Vaultwarden)
Server Version 2.25.1 

Client linux Fedora, rpm package.
Version 2023.2.0

Everything works fine in android client.
![изображение](https://user-images.githubusercontent.com/34908221/224823733-6a92fe7f-1b38-4c34-81ae-76bd3f290fa5.png)

**Comment by BlackDex** [comment, marked as answer]
You are using an incompatible web-vault with the server version.
You need the main branch or testing version.

**Comment by aZZmNZtA** [comment]
Добрый вечер! @boodmi   
Подскажите, пожалуйста, решили вопрос с входом? 
Та же проблема. 
У меня для chrome расширение не входит.

Server Installed  1.24.0
Server Latest 1.27.0
Web Installed 2.25.1

Пытаюсь понять какую версию ставить чтобы работало.
https://github.com/bitwarden/clients/releases?page=2

Как вы решили проблему?
Спасибо заранее!
